configure

```diff
@@ -2026,6 +2026,8 @@ ac_configure="$SHELL $ac_aux_dir/configure"  # Please don't use this var.
 
 
 
+: ${CFLAGS=""}
+: ${CXXFLAGS=""}
```

Comment:
Did you regenerate this script? Or just put these lines manually? I see in docs that this syntax is required in configue.ac. But should it come here in the same form?

Reply:
>Did you regenerate this script? 

Yes.

>Or just put these lines manually?

No.